rsmtool/rsmsummarize.py

```diff
@@ -129,15 +139,29 @@ def run_summary(config_file_or_obj, output_dir):
         else:
             configpath = os.getcwd()
 
+    logger.info('Saving configuration file.')
+    configuration.save(output_dir)
+
     # get the list of the experiment dirs
     experiment_dirs = configuration['experiment_dirs']
 
+    # Get experiment names if any
+    experiment_names = configuration.get('experiment_names')
+    if experiment_names:
+        dirs_with_names = zip(experiment_dirs, experiment_names)
+    else:
+        dirs_with_names = zip(experiment_dirs, [None]*len(experiment_dirs))
```

Comment:
Just a thought, it might be a bit more readable to rewrite this block as:

```python
experiment_names = configuration.get('experiment_names')
experiment_names = experiment_names if experiment_names else [None]*len(experiment_dirs)
dirs_with_names = zip(experiment_dirs, experiment_names)
```